[2024-11-17T15:39:59.657+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2024-11-17T15:40:00.245+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: add_data_API.run_API_news scheduled__2024-11-17T06:00:00+00:00 [queued]>
[2024-11-17T15:40:00.318+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: add_data_API.run_API_news scheduled__2024-11-17T06:00:00+00:00 [queued]>
[2024-11-17T15:40:00.403+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 2
[2024-11-17T15:40:00.589+0000] {taskinstance.py:2888} INFO - Executing <Task(PythonOperator): run_API_news> on 2024-11-17 06:00:00+00:00
[2024-11-17T15:40:00.634+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=937) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-11-17T15:40:00.647+0000] {standard_task_runner.py:72} INFO - Started process 950 to run task
[2024-11-17T15:40:00.702+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'add_data_API', 'run_API_news', 'scheduled__2024-11-17T06:00:00+00:00', '--job-id', '689', '--raw', '--subdir', 'DAGS_FOLDER/insert_data.py', '--cfg-path', '/tmp/tmp8_qv_be7']
[2024-11-17T15:40:00.712+0000] {standard_task_runner.py:105} INFO - Job 689: Subtask run_API_news
[2024-11-17T15:40:01.164+0000] {task_command.py:467} INFO - Running <TaskInstance: add_data_API.run_API_news scheduled__2024-11-17T06:00:00+00:00 [running]> on host c967fb0415e1
[2024-11-17T15:40:01.432+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='add_data_API' AIRFLOW_CTX_TASK_ID='run_API_news' AIRFLOW_CTX_EXECUTION_DATE='2024-11-17T06:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-11-17T06:00:00+00:00'
[2024-11-17T15:40:01.436+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2024-11-17T15:40:01.437+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2024-11-17T15:40:01.438+0000] {logging_mixin.py:190} INFO - Current task name:run_API_news state:running start_date:2024-11-17 15:40:00.259241+00:00
[2024-11-17T15:40:01.439+0000] {logging_mixin.py:190} INFO - Dag name:add_data_API and current dag run status:running
[2024-11-17T15:40:01.440+0000] {taskinstance.py:731} INFO - ::endgroup::
[2024-11-17T15:40:01.640+0000] {logging_mixin.py:190} INFO - Erreur lors de l'ex√©cution du script : Command '['python3', 'C:/Users/33672/OneDrive/Documents/GitHub/Projet_EDD/script/getAPI.py']' returned non-zero exit status 2.
[2024-11-17T15:40:01.642+0000] {python.py:240} INFO - Done. Returned value was: None
[2024-11-17T15:40:01.683+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2024-11-17T15:40:01.684+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=add_data_API, task_id=run_API_news, run_id=scheduled__2024-11-17T06:00:00+00:00, execution_date=20241117T060000, start_date=20241117T154000, end_date=20241117T154001
[2024-11-17T15:40:01.708+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2024-11-17T15:40:01.709+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2024-11-17T15:40:01.710+0000] {logging_mixin.py:190} INFO - Dag name:add_data_API queued_at:2024-11-17 15:36:06.373727+00:00
[2024-11-17T15:40:01.711+0000] {logging_mixin.py:190} INFO - Task hostname:c967fb0415e1 operator:PythonOperator
[2024-11-17T15:40:01.756+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2024-11-17T15:40:01.803+0000] {taskinstance.py:3900} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-11-17T15:40:01.818+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
